src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -16,45 +16,45 @@
 import frc.robot.Constants.TurretConstants;
 
 public class TurretSubsystem extends SubsystemBase {
-  private WPI_TalonFX turret = new WPI_TalonFX(TurretConstants.kTurretMotorPort);
+  private WPI_TalonFX this.turret = new WPI_TalonFX(TurretConstants.kTurretMotorPort);
```

Comment:
great work! however here–you don't need to address it as "this.turret" because it is not officially declared yet. once it is declared the class's field, that is when you address it as "this.turret"

Reply:
Ok, should now be fixed here too.